README.md

```diff
@@ -3,148 +3,88 @@
 [![Build Status](https://travis-ci.org/PaddlePaddle/cloud.svg?branch=develop)](https://travis-ci.org/PaddlePaddle/cloud)
 
 PaddlePaddle Cloud is a Distributed Deep-Learning Cloud Platform for both cloud
-providers and enterprises.
+providers and enterprises. PaddlePaddle Cloud provide cluster Deep-Learning
+features including:
+
+- Manage cluster Deep-Learning jobs use a command-line client very easily.
+- Elastic Deep-Learning ([EDL](./doc/edl/README.md)) enables maximizing cluster resource utilities
+  and reduce average job pending time.
+- Manages thousands of GPUs in cluster.
+- Fault tolerant distributed training with zero down time.
 
 PaddlePaddle Cloud use [Kubernetes](https://kubernetes.io) as it's backend job
 dispatching and cluster resource management center. And use [PaddlePaddle](https://github.com/PaddlePaddle/Paddle.git)
-as the deep-learning frame work. Users can use web pages or command-line tools
-to submit their deep-learning training jobs remotely to make use of power of
-large scale GPU clusters.
-
-## Using Command-line To Submit Cloud Training Jobs
+as the deep-learning frame work. 
+
+## Components
+
+- Server side components includes:
+  - Cloud Server
+    A REST API server accepts incomming request from lommand-line client, and submit
+    `TrainingJob` resource to Kubernetes cluster. Cloud Server is written using
+    `Django` framework under `python` directory.
```

Comment:
Cloud you add a link for the python directory.

Reply:
I use directory names in this section to prevent link change, detailed directory description can be found at the bottom.